src/style.css

```diff
@@ -1,3 +1,83 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+header {
+  height: 60px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  font-family: Roboto, sans-serif;
```

Comment:
```suggestion

```
It's better to specify the font family for the body tag to get it inherited by other elements